common/djangoapps/student/tests/test_email.py

```diff
@@ -62,6 +64,72 @@ def append_allowed_hosts(self, hostname):
         self.addCleanup(settings.ALLOWED_HOSTS.pop)
 
 
+@unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Test only valid in lms')
+class ActivationEmailTests(TestCase):
+    """Test sending of the activation email. """
+
+    ACTIVATION_SUBJECT = "Activate Your edX Account"
+
+    # Text fragments we expect in the body of an email
+    # sent from an OpenEdX installation.
+    OPENEDX_FRAGMENTS = [
+        "Thank you for signing up for edX.",
+        "http://edx.org/activate/",
+        "if you require assistance, check the help section of the edX website"
+    ]
```

Comment:
These fragments would be more robust if `settings.PLATFORM_NAME` was used in place of `edX`.